base/simplesaml/ansible/inv/local/group_vars/all.tmpl

```diff
@@ -26,3 +26,6 @@ simplesaml_salt: ${SIMPLESAML_SALT}
 # The DB credentials
 db_username: ${DB_USERNAME}
 db_password: ${DB_PASSWORD}
+
+# The root password needed to install mysql users
+mysql_root_password: ${MYSQL_ROOT_PASSWORD}
```

Comment:
It might not make any functional difference, but having a newline at the end of the file would help various tools not complain, is editor-macro-friendly, etc.

Reply:
Hmm, I'm used to my editor inserting those... I wonder why it didn't this time.